### PR TITLE
Add instruction for presto benchto driver data bootstrapping 

### PR DESCRIPTION
**Pull request by willshen**
Adding more instruction for presto benchto driver data bootstrapping to clarify what needs to be done before the benchmark driver can be run.

### CODE REVIEW

**Comment by willshen** [comment]
Actually, realized that this is not correct.

**Comment by willshen** [comment]
Reopening. This should be correct after we also bring in https://github.com/prestosql/presto/pull/1082

**Comment by willshen** [comment]
@cla-bot check

File: presto-benchto-benchmarks/README.md [hunk 1]
```diff
@@ -62,6 +62,17 @@ macros:
       
 ```
 
+### Bootstrapping benchmark data
+* Configure Presto [TCP-H connector](https://prestosql.io/docs/current/connector/tpch.html), if you haven't.
+* Generate SQL file:
+```bash
+python presto-benchto-benchmarks/generate_schemas/generate-tpch.py tpch.sql
```
Comment (findepi) [inline presto-benchto-benchmarks/README.md]: nit: It should be possible to pipe this directly to presto-cli

File: presto-benchto-benchmarks/README.md [hunk 2]
```diff
@@ -62,6 +62,17 @@ macros:
       
 ```
 
+### Bootstrapping benchmark data
+* Configure Presto [TCP-H connector](https://prestosql.io/docs/current/connector/tpch.html), if you haven't.
```
Comment (findepi) [inline presto-benchto-benchmarks/README.md]:
```suggestion
* Make sure you have configured [Presto TCP-H connector](https://prestosql.io/docs/current/connector/tpch.html).
```

Comment (willshen) [reply presto-benchto-benchmarks/README.md]: good idea!

File: presto-benchto-benchmarks/README.md (hunk 1 above)
Comment (willshen) [reply presto-benchto-benchmarks/README.md]: good idea!

**Comment by findepi** [comment]
Merged as 4b62d406bb302723616d16e02b784c3967e28598, thanks!
(I squashed the commits and did a formatting fix.)

**Pull request closed without merging**